src/abstractdataframe/iteration.jl

```diff
@@ -19,13 +19,9 @@ which is a view that acts like a one-row DataFrame.
 eachrow(df::AbstractDataFrame) = DFRowIterator(df)
 
 function Base.iterate(itr::DFRowIterator, i=1)
-    if i > size(itr.df, 1)
-        nothing
-    else
-        (DataFrameRow(itr.df, i), i + 1)
-    end
+    i > size(itr.df, 1) && return nothing
+    return (DataFrameRow(itr.df, i), i + 1)
```

Comment:
This is fine, but isn't it the same as what's already there? Not sure if I'm missing something.

Reply:
It is, just more compact; I actually did this before @bkamins, and I resolved the conflict to my version.